internal/generator/fluentd/outputs.go

```diff
@@ -40,6 +40,26 @@ func Outputs(clspec *logging.ClusterLoggingSpec, secrets map[string]*corev1.Secr
 			outputs = MergeElements(outputs, loki.Conf(bufspec, secret, o, op))
 		}
 	}
-
 	return outputs
 }
+
+// GetAppendVolumes Append any additional volumes based output type and values from secret and forwarder spec
+// otherwise return empty
+func GetAppendVolumes(secrets map[string]*corev1.Secret, clfspec *logging.ClusterLogForwarderSpec) (mount corev1.VolumeMount, volume corev1.Volume) {
```

Comment:
Same comment here.  This is not generator code but podspec code.  We should find a home for it there